DEVELOPMENT.md

```diff
@@ -102,7 +109,8 @@ Once you reach this point you are ready to do a full build and deploy as describ
 ## Starting Knative Serving
 
 Once you've [setup your development environment](#getting-started), stand up
-`Knative Serving`:
+`Knative Serving`. Note that if you already installed Knative to your cluster,
+you need to [clean it up](#clean-up) first before following these instructions.
```

Comment:
I'm not sure the cleanup is needed; you should just be able to replace items a la iterative development.

Reply:
Done